pallets/omnipool/src/lib.rs

```diff
@@ -223,6 +219,10 @@ pub mod pallet {
 	/// Position ids sequencer
 	pub(super) type NextPositionId<T: Config> = StorageValue<_, T::PositionItemId, ValueQuery>;
 
+	#[pallet::storage]
+	/// TVL cap
+	pub(super) type TvlCap<T: Config> = StorageValue<_, Balance, ValueQuery>;
```

Comment:
I would either make this an option query indicating that it can be unset (and define the default at call time) or set a reasonable default value.

Reply:
0 is a reasonable default value as it prevents anyone to add liquidity

Reply:
I was thinking to make it OptionQuery. 

and since we dont want to have default value as constant somewhere. this would introduce new error too ( in case of no tvl cap set ). 

I am not sure yet about this one. 

so far, i am for default value to be 0. no harm, since you can't add liquidity at all. and it does not have impact on other functionality.

Reply:
MAX would be also acceptable, but less safe in my opinion